### PR TITLE
Features/1008 gae mail attachment to gcs

### PR DESCRIPTION
**Pull request by smpiano**
Read email attachments a store them in the GCS indicated in the config folder.

Related with> https://github.com/GlobalFishingWatch/GFW-Tasks/issues/1008
Notes> https://docs.google.com/document/d/1ZF4lrxS9A4O4_BcVVMdIokNCsx9kPaPyTWlf8YgH2jc/edit?usp=sharing

### CODE REVIEW

File: config/reception.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[GCS]
+BUCKET=scratch-matias
+DIRECTORY=panama/attachments
+BOTO_PARALLEL_PROCESS=76
```
Comment (andres-arana) [inline config/reception.cfg]: Do you really need the BOTO configuration options here? I thought these are only required for `gsutil`, which we aren't using now.

File: utils/upload_gcs.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+class GCSTransfer:
+    def __init__(self):
+        #This line fixes the error
+        #AttributeError: 'VerifiedHTTPSConnection' object has no attribute '_tunnel_host'
+        appengine.monkeypatch()
+        configParser = ConfigParser.RawConfigParser()
+        file = r'./config/reception.cfg'
+        configParser.read(file)
+        self.boto_process = configParser.get('GCS', 'BOTO_PARALLEL_PROCESS')
```
Comment (andres-arana) [inline utils/upload_gcs.py]: Again, these boto settings were required for the `gsutil` approach, which we are not using anymore. Remove this.

**Review by andres-arana — CHANGES_REQUESTED** [review]
Just some quick changes regarding old code from when we tried the `gsutil cp` approach.

**Review by enriquetuya — CHANGES_REQUESTED** [review]
@smpiano we will start receiving mails from 4 other mail accounts with attachments with the same format and same periodicity (or maybe even more) as Panama.
Let's make sure we have the code to support that.

**Pull request merged**